### DISCUSSION TITLE
Jump Drive seems far too cheap

### DISCUSSION THREAD

**Discussion by opusforlife2** [endless-sky/endless-sky#10633] [Lore]
Just a million bucks to be able to _completely bypass all hyperlane limitations_ seems like far too little a price, from a lore perspective. You could get that by selling 4 Hawks. It should cost far, _far_ more than that. Like a 100 million.

However, that would make Jump Drive farming insanely lucrative. So, maybe it should just show a price of `???` while the selling price remains the same, since it's essentially unknown tech. Who knows? Maybe the Korath are able to manufacture it for just 10,000 bucks. Or it could be a cool billion.

A similar thing would apply to stuff like the Cloaking Device, and any other unique outfits. Their price should be `???`. If it's unpurchasable anywhere, it shouldn't show a definite price.

**Comment by Saugia** [comment]
I feel this is one of those examples where gameplay > lore perspective. 1 million is a *lot* of money for presumably a majority of humans, considering that owning a ship in the first place is a privilege in itself. Humans are also somewhat stingy, after all.

It really comes down to how specific we want to be about it, and I think the more picky the kind of worse off we get. The Uhai basically infinitely trade you 1mil per JD - you can farm JDs, and make bank, so increasing the money would be a pretty big balance concern. If you can give the Uhai unlimited JDs, why haven't they taken over the galaxy? Why aren't the Quarg concerned about what ability you have here? It can end up a rabbit hole pretty fast.

Same logic with the (???). We hire crew on random planets and credits are the same between species. That's intentional for gameplay ease, rather than nuanced for lore reasons. At this point, if you still get 1mil from selling it, just keep it 1mil in the outfitter. This is what the *player* is seeing anyway, it doesn't necessarily mean that the JD is universally understood by npcs in the ES universe to be worth 1 million.

**Reply by opusforlife2** [reply]
The Remnant buy broken Jump Drives off of you for a price of 300,000, despite the outfitter showing they cost 25000 (in lore that would just be people using it for scrap metal or something) . A similar thing could be done with the U-Hai. The actual price of Jump Drives doesn't matter.
